src/contract.rs

```diff
@@ -58,10 +60,12 @@ fn create_transfer(
         recipient: deps.api.addr_validate(&recipient)?,
     };
 
+    let querier = MarkerQuerier::new(&deps.querier);
+
     let is_restricted_marker = matches!(
-        ProvenanceQuerier::new(&deps.querier).get_marker_by_denom(transfer.denom.clone()),
-        Ok(Marker {
-            marker_type: MarkerType::Restricted,
+        get_marker_by_denom(transfer.denom.clone(), &querier),
+        Ok(MarkerAccount {
+            marker_type: 2, // MarkerType::Restricted,
```

Comment:
Don't like the magic number here, but trying to `MarkerType::Restricted` or `MarkerType::Restricted as i64` doesn't work.

Reply:
i had the same issue https://github.com/FigureTechnologies/digital-currency-consortium/pull/159/files#diff-05a53b4776df07c901df8f7a44dce40def88abdd8a4f9e5395c3de40f3bb2b61R63